Build fails due to API change

From this PR: edgexfoundry/edgex-go#724 a breaking API change was introduced which causes this project to no longer compile. See the delhi jenkins jobs for example: https://jenkins.edgexfoundry.org/view/security/job/security-secret-store-delhi-stage-go/1/console

```
00:49:09 + make build
00:49:09 cd pkisetup && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags="-s -w" -o pkisetup .
00:49:24 cd core && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o edgex-vault-worker .
00:49:51 # github.com/edgexfoundry/security-secret-store/core
00:49:51 ./main.go:21:25: not enough arguments in call to logger.NewClient
00:49:51 	have (string, bool, string)
00:49:51 	want (string, bool, string, string)
00:49:51 make: *** [build] Error 2
00:49:51 Build step 'Execute shell' marked build as failure

```
I propose that we use a default logging level of INFO (i.e. logger.InfoLog) as the third argument here: https://github.com/edgexfoundry/security-secret-store/blob/master/core/main.go#L21.

--- a/core/main.go
+++ b/core/main.go
@@ -18,7 +18,7 @@ var lc = CreateLogging()
 
 // CreateLogging Logger functionality
 func CreateLogging() logger.LoggingClient {
-	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")))
+	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")), logger.InfoLog)
 }
 
 func main() {

--- a/glide.yaml
+++ b/glide.yaml
@@ -9,5 +9,6 @@ import:
 - package: github.com/hashicorp/vault/api
   version: v0.11.3  
 - package: github.com/edgexfoundry/edgex-go
+  version: delhi
   subpackages:
   - pkg/clients/logging

